analysis/combiners.py

```diff
@@ -160,14 +161,27 @@ def _compute_pmf(self) -> poisson_binomial.PMF:
                                       Optional[SumOfRandomVariablesMoments]]
 
 
+def _combine_list(a: List, b: List) -> List:
```

Comment:
Optional: WDYT of `merge` instead of `combine`? That is what we use to describe the same operation for DP Building Blocks primitives.